pages/common/ld.md

```diff
@@ -0,0 +1,16 @@
+# ld
+
+> The GNU ELF linker.
+> More information: <https://ftp.gnu.org/old-gnu/Manuals/ld-2.9.1/html_mono/ld.html>.
+
+- Link an object file with no dependencies into an executable:
```

Comment:
```suggestion
- Link the object file with no dependencies into the executable:
```

Reply:
No, this reads better with `an`, as the executable in question is not a specific executable - this is a generic case of an action.